query-engine/core/src/query_graph_builder/read/many.rs

```diff
@@ -1,8 +1,22 @@
 use super::*;
-use crate::{query_document::ParsedField, ManyRecordsQuery, ReadQuery};
+use crate::{query_document::ParsedField, ManyRecordsQuery, QueryOption, ReadQuery};
+use enumflags2::BitFlags;
 use prisma_models::ModelRef;
 
 pub fn find_many(field: ParsedField, model: ModelRef) -> QueryGraphBuilderResult<ReadQuery> {
+    find_many_with_options(field, model, BitFlags::EMPTY)
+}
+
+pub fn find_many_or_throw(field: ParsedField, model: ModelRef) -> QueryGraphBuilderResult<ReadQuery> {
+    find_many_with_options(field, model, BitFlags::from_flag(QueryOption::ThrowOnEmpty))
```

Comment:
Same here, we leaking the BitFlag abstraction here. Is there another way around this. We want to use BitFlags internally but not expose that outside the QueryOption. 

you might have to have something like:

```
struct QueryOption {
   inner: BitFlags
   
}
```